boards/arm/beagle_play_cc1352/doc/index.rst

```diff
@@ -0,0 +1,179 @@
+.. _beagleplay_cc1352:
+
+BeaglePlay (CC1352)
+###################
+
+Overview
+********
+
+BeagleBoard.org BeaglePlay is an open hardware single board computer based on a TI Sitara AM6254
+quad-core ARM Cortex-A53 SoC with an external TI SimpleLink multi-standard CC1352P7 wireless MCU
+providing long-range, low-power connectivity.
+
```

Comment:
can you provide an image? It's always nice to have while not 100% mandatory. Bonus point for having it in webp format, and possibly with transparent background. See ex. https://docs.zephyrproject.org/latest/boards/arm/beagle_bcf/doc/index.html